lab_5_scrapper/scrapper.py

```diff
@@ -120,46 +254,96 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(url=full_url, article_id=article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        texts_bs = article_soup.find('div', {
+            'class':
+                'elementor-element elementor-element-6732342 elementor-widget elementor-widget-theme-post-content'})
+        texts = texts_bs.find_all('p')
+        result = ''
+        for el in texts:
+            txt = el.text
+            txt = txt.replace('\n\t\t\t\t', '')
+            txt = txt.replace('\t\t\t', '')
+            if txt != '\xa0':
+                result = ' '.join([result, txt])
+        self.article.text = result
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        title_bs = article_soup.find('h1', {'class': 'elementor-heading-title elementor-size-default'})
+        self.article.title = title_bs.text
+
+        date_bs = article_soup.find('span',
+                                    {'class': 'elementor-icon-list-text elementor-post-info__item elementor-post-info__item--type-date'})
+        date_txt = re.search(r'\d{2}/\d{2}/\d{4}', date_bs.text)
+        self.article.date = date_txt[0]
+
+        auth_bs = article_soup.find('span', {
+            'class': 'elementor-icon-list-text elementor-post-info__item elementor-post-info__item--type-author'})
+        auth_txt = re.search(r'\w+\s\w+', auth_bs.text)
+        self.article.author = auth_txt[0]
```

Comment:
author should be a list

Reply:
do you mean rewriting it to `self.article.author = [auth_txt[0]]`?